src/main/java/com/gmail/nossr50/util/skills/SkillUtils.java

```diff
@@ -266,6 +266,36 @@ private static boolean isLocalizedSkill(String skillName) {
 
         return false;
     }
+    
+    
+    /**
+     * Modify the durability of an ItemStack, using Armor specific formula for unbreaking enchant damage reduction
+     *
+     * @param itemStack The ItemStack which durability should be modified
+     * @param durabilityModifier the amount to modify the durability by
+     * @param maxDamageModifier the amount to adjust the max damage by
+     */
+    public static void handleArmorDurabilityChange(ItemStack itemStack, double durabilityModifier, double maxDamageModifier) {
+        if(itemStack.getItemMeta() != null && itemStack.getItemMeta().isUnbreakable()) {
+            return;
+        }
```

Comment:
Also just a side note that `ItemStack#getItemMeta()` is a **very** heavy call to perform.
So the value should be stored as a variable for performance improvements.

*(The same improvement can of course also be applied to the original method)*

```suggestion
        ItemMeta itemMeta = itemStack.getItemMeta();
        if (itemMeta  != null && itemMeta.isUnbreakable()) {
            return;
        }
```

Reply:
itemStack.getItemMeta() is always !=null, we should just use itemStack.hasItemMeta()

Reply:
Technically not. `.getItemMeta()` is explicitly marked as `@Nullable`.
The method's contract does allow null to be returned.

However no current implementation ever returns a null value, so it practically is never null.
But yes, I agree, either way `ItemStack#hasItemMeta()` would be the optimal approach regardless.